src/makeCalculator.js

```diff
@@ -38,6 +38,43 @@
  */
 function makeCalculator() {
   // write code here
+  const calculatedResult = {
+    result: 0,
+
+    add(a, b) {
+      return a + b;
+    },
+
+    subtract(a, b) {
+      return a - b;
+    },
+
+    multiply(a, b) {
+      return a * b;
+    },
+
+    divide(a, b) {
+      if (b === 0) {
+        throw new Error(`${b} = 0`);
+      }
```

Comment:
При такой записи пользователю будет выводиться ошибка "0 = 0", думаю, немного сложно будет понять, что именно не так)
Возможно лучше выводить сообщение с описанием ошибки, например:
```suggestion
      if (b === 0) {
        throw new Error(`Invalid value`);
      }
```